lib/utils.js

```diff
@@ -35,6 +36,65 @@ async function getSimpleCypressConfig(config) {
   };
 }
 
+async function convertJsonScreenshotsToBase64(jsonsDir,screenshotsDir) {
+  const dirResult = await fs.readdirSync(jsonsDir);
+  const jsonParsed = [];
+
+  for (let i = 0; i < dirResult.length; i++) {
+    const element = dirResult[i];
+    const filePath = path.resolve(jsonsDir,element);
+    jsonParsed.push({
+      file: filePath, data: await fse.readJson(filePath)
+    });
+  }
+
+  function convertContextImgToBase64(contextObj){
+    if (contextObj.value) {
```

Comment:
The user can also add context, so we need to verify that we don't change user context
`context` can also be a simple string without `title` & `value`.
let's validate `context` is an object & the value starts with `screenshots`

```suggestion
if (typeof context === 'object' && context.value && context.title && context.value.startsWith('screenshots')) {```